Sources/TABResourceLoader/Operations/BaseAsynchronousOperation.swift

```diff
@@ -62,12 +63,14 @@ open class BaseAsynchronousOperation: Operation {
     execute()
   }
 
+  /// Method to be overriden with the implemenation of the work done on the operation
   open func execute() {
     assertionFailure("execute must overriden")
     finish()
   }
 
-  public final func finish(_ errors: [Error] = []) {
+  /// Call this method after all the work is done signal the completion of this operation
```

Comment:
The idea is that this method is called to finished the execution of the operation. I think I for the word to is missing, probably should say:

Call this method after all the work is done to signal the completion of this operation

Does that make it clearer? Otherwise maybe we can rephrase the doc altogether since it's quite important that this concept is understood.